internal/proxy/impl.go

```diff
@@ -2774,12 +2774,143 @@ func (node *Proxy) CreateAlias(ctx context.Context, request *milvuspb.CreateAlia
 	return cat.result, nil
 }
 
+// DescribeAlias describe alias of collection.
 func (node *Proxy) DescribeAlias(ctx context.Context, request *milvuspb.DescribeAliasRequest) (*milvuspb.DescribeAliasResponse, error) {
-	panic("TODO: implement me")
+	if !node.checkHealthy() {
+		return &milvuspb.DescribeAliasResponse{
+			Status: merr.Status(merr.WrapErrServiceNotReady(fmt.Sprintf("proxy %d is unhealthy", node.session.ServerID))),
+		}, nil
+	}
+
+	ctx, sp := otel.Tracer(typeutil.ProxyRole).Start(ctx, "Proxy-DescribeAlias")
+	defer sp.End()
+
+	dat := &DescribeAliasTask{
```

Comment:
How about using `return node.rootCoord.DescribeAlias` directly? Anyway, these two apis will be scheduled by the rootcoord's queue.